nvm.sh

```diff
@@ -1965,6 +1968,13 @@ nvm_install_binary() {
     return 0
   fi
 
+
+  # Read nosource from arguments
+  if [ "$nosource" -eq 1 ]; then
```

Comment:
```suggestion
  if [ "${nosource-}" = '1' ]; then
```
i think this one's a string at this point